Rakefile

```diff
@@ -11,7 +11,7 @@ task :console do
   if File.exist? config_file
     config = YAML.load File.read(config_file)
 
-    AWS.config(config['aws'])
+    Aws.config = config['aws']
```

Comment:
@leemhenson what do you do with shoryuken using `rails console` just `Worker.perform_async`?